File: TransCoder_tokenized_test_set_functions/java/CHECK_LARGE_NUMBER_DIVISIBLE_20.java
static Boolean divisibleBy20 ( String num ) {
  int lastTwoDigits = Integer . parseInt ( num . substring ( num . length ( ) - 2, num . length ( ) ) );
  return ( ( lastTwoDigits % 5 == 0 ) && ( lastTwoDigits % 4 == 0 ) );
}

File: TransCoder_tokenized_test_set_functions/cpp/CHECK_LARGE_NUMBER_DIVISIBLE_20.cpp
bool divisibleBy20 ( string num ) {
  int lastTwoDigits = stoi ( num . substr ( num . length ( ) - 2, num . length ( ) - 1 ) );
  return ( ( lastTwoDigits % 5 == 0 ) && ( lastTwoDigits % 4 == 0 ) );
}